zircon-loader/src/lib.rs

```diff
@@ -206,6 +206,9 @@ async fn new_thread(thread: CurrentThread) {
         thread.time_add(time);
         trace!("back from user: {:#x?}", cx);
         EXCEPTIONS_USER.add(1);
+        let cx_clone = cx.clone();
```

Comment:
Cloning the context is expensive. Should be avoided?

Reply:
But how?